drivers/storage/portworx/component/prometheus.go

```diff
@@ -571,7 +571,7 @@ func (c *prometheus) createPrometheusInstance(
 		cluster.Spec.Monitoring.Prometheus.RemoteWriteEndpoint != "" {
 		prometheusInst.Spec.RemoteWrite = []monitoringv1.RemoteWriteSpec{
 			{
-				URL: fmt.Sprintf("http://%s/api/prom/push", cluster.Spec.Monitoring.Prometheus.RemoteWriteEndpoint),
+				URL: fmt.Sprintf("%s/api/prom/push", cluster.Spec.Monitoring.Prometheus.RemoteWriteEndpoint),
```

Comment:
does it get redirected according to the protocol, i.e 80 for HTTP and 443 for https?
how does prometheus figure it out?

Reply:
will set this protocol in endpoint form pxcentral api-server, as per this: https://github.com/portworx/pxcentral-onprem/pull/35/files#diff-202b2517a7852bf6ae601a7c789581917fdded59b8758165cef13993e852ca6cR517-R520